gatsby-config.ts

```diff
@@ -18,6 +18,7 @@ const gatsbyConfig: GatsbyConfig = {
   },
   flags: {
     FAST_DEV: true,
+    DEV_SSR: false,
```

Comment:
out of curiosity: why is it false?

> Now you can add the DEV_SSR flag to your gatsby-config file so you can spot and fix SSR errors (like trying to access the window object) during development.


https://www.gatsbyjs.com/docs/reference/release-notes/v4.18/#notable-bugfixes--improvements:~:text=spot%20and%20fix%20SSR%20errors

Reply:
when I enable devSSR everything breaks with a hydration mismatch error. Somehow Gatsby does not render the page.tx, only the layout.tsx, creating this mismatch